Update cobbler dhcp to include EFI PXE example (#697)
uyuni-project/uyuni-docs: modules/client-configuration/pages/cobbler.adoc

The PXE boot process uses DHCP to find the TFTP boot server.
The {productname} Server can act as such a  TFTP boot server and Cobbler can generate the content for it.

> DHCP server can be configured directly or using DHCPd formula.

Reviewer: ```suggestion
The DHCP server can be configured directly, or you can use the DHCPd formula.
```

To configure DHCP directly, you must have administrative access to the network's DHCP server.
Edit the DHCP configuration file so that it points to the {productname} Server as the TFTP boot server:

> .Procedure: Example for Configuring the ISC DHCP Server

Reviewer: ```suggestion
.Procedure: Configuring the ISC DHCP Server Directly
```

Just to match it back to the language we use above.

* Creates a class called ``PXE``.
* Identifies systems as ``PXEClient`` if they are configured with PXE as the first boot priority.
* Directs PXEClients to the Cobbler server at ``192.168.2.1``.
* Retrieves the [path]``grub/grub.efi`` bootloader file for EFI PXE clients.
* Retrieves the [path]``pxelinux.0`` bootloader file for legacy BIOS PXE clients.
+
. Save the file.

> .Procedure: Example for configuring ISC DHCP Server using DHCPd formula

Reviewer: ```suggestion
.Procedure: Configuring an ISC DHCP Server with the DHCPd formula
```

Author: I am not sure if this will not confuse people. There are two examples - manual configuring and this one, using dhcpd formula - and its up to user which one, but only one, they will use.

Reviewer: In that case, we should probably include some explanation to that effect in a para before the procedure.

Author: I added some explaining text above.

> . Configure DHCPd formula as described in xref:salt:formula-dhcpd.adoc[]

Reviewer: ```suggestion
. Configure the DHCPd formula.
  For more information, see xref:salt:formula-dhcpd.adoc[]
```

> . Use [path]``grub/grub.efi`` in [guimenu]``Filename EFI`` field for EFI PXE support
> . Use [path]``pxelinux.0`` in [guimenu]``Filename`` field for legacy BIOS support

Reviewer: ```suggestion
. In the [guimenu]``Filename EFI`` field, for EFI PXE support type [path]``grub/grub.efi``.
  Alternatively, for legacy BIOS support type [path]``pxelinux.0``.
```

Author: This is not exactly what I meant. I mean that user should put both values there to support both legacy and efi PXE. So that this example will result in same behaviour as the manual example above.

Reviewer: Ah, I understand, let's go back to two steps then, assuming they are two different fields. How about this:

```suggestion
. In the  [guimenu]``Filename EFI`` field, type [path]``grub/grub.efi`` to enable EFI PXE support.
. In the  [guimenu]``Filename`` field, type [path]``pxelinux.0`` to enable legacy BIOS support.
```

* Creates a class called ``PXE``.
* Identifies systems as ``PXEClient`` if they are configured with PXE as the first boot priority.
* Directs PXEClients to the Cobbler server at ``192.168.2.1``.
* Retrieves the [path]``grub/grub.efi`` bootloader file for EFI PXE clients.
* Retrieves the [path]``pxelinux.0`` bootloader file for legacy BIOS PXE clients.
+
. Save the file.

> To configure DHCP using DHCPd formula, follow this example:

Reviewer: ```suggestion
Alternatively, you can configure DHCP using the DHCPd formula instead:
```
